README.md

```diff
@@ -106,7 +106,7 @@ For this quick start, we'll skip over most of the features, but be sure to take
 
 Here is an easy example for serving an object classification model:
 ```bash
-mxnet-model-server --models squeezenet=https://s3.amazonaws.com/model-server/model_archive_1.0/squeezenet_v1.1.mar
+mxnet-model-server --start --models squeezenet=https://s3.amazonaws.com/model-server/model_archive_1.0/squeezenet_v1.1.mar
```

Comment:
Is there a strong reason to add this back? --start currently doesn't do anything. Without this, the CLI is simple and also keeps in tune with the previous version of MMS.

Reply:
1. with --start is more explicit to user
2. it's make it easy for user to aware there is --stop to stop the model-server, discover --stop become important since we are start mms as background process now.

Reply:
So... my 2 cents on this is to introduce a pattern of:
1. start mms
2. invoke model loading
3. infer test
4. stop

Then you can introduce shortcuts that combine 1-2. And when you do so, you're consistent. I'd leave --start in there, regardless of it triggering things or not.

Also if you guys went with my suggestion of having `mxnet-model-server` command on its own (no --start) outputs config and status (if it is running), then for the user, this all makes sense.